CONTRIBUTING.md

```diff
@@ -9,25 +9,29 @@
 * [Idle Pull Requests Policy](#idle)
 * [Unresolved Pull Requests Policy](#unresolved)
 
+This document is an extension to the generic [Anuket Project Operations and Procedures](https://wiki.anuket.io/display/HOME/Anuket+Project+Operations+and+Procedures) and describes contribution rules to the Anuket specifications, that are the set of documents maintained in this repository. 
+
 <a name="permissions"></a>
 ### Permission levels
-There are different permissions (levels) available for contributing into CNTT:
+
+There are different permissions (levels) available for contributing into the Anuket specifications:
 - **Outside Collaborators**: This includes general public. 
   - Outside collaborators can create issues and add comments to issues and Pull Requests.
 - **Read, Triage, Write, and Maintain** Permissions, which are explained in [here](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization), will be assigned as follows:
-  - **Read** will be assigned to those who are interested to create issues, join the discussion, receive notifications, and provide feedback.
+  - **Read** will be assigned to those who want to create issues, join the discussions, receive notifications, and provide feedback.
   - **Triage** will be assigned to active contributors who create issues, pull requests and review them.
-  - **Writes** will be assigned to Workstream Co-Leads to allow them to manage issues and pull request, approve them and be auto requested for review.
-  - **Maintain** will be assigned to Workstream Leads to allow them to merge PRs into the repository.
+  - **Writes** will be assigned to sub-project Co-Leads to allow them to manage issues and pull request, approve them and be auto requested for review.
+  - **Maintain** will be assigned to sub-project Leads to allow them to merge PRs into the repository.
 
 <a name="labels"></a>
 ### Labels
-Every Issue and PR should be tagged with one or more of the following labels to indicate their status and for the automation pipelines to apply the correct CNTT processes.
+
+Every Issue and PR should be tagged with one or more of the following labels to indicate their status and for the automation pipelines to apply the correct Anuket processes.
 
   - **Backlog**: The Issue will be addressed in future releases..
   - **Major/Minor**: The Issue or PR concerns a major or minor change.. 
-  - **WIP**: The PR is still being developed and no reviews/approvals are expected yet.
-     - PRs can also be created as "Draft" indicate that the PR is working in progress.
+  - **WIP**: The PR is still under development, no reviews/approvals are expected yet.
+     - PRs can also be created as "Draft" to indicate that the PR is in progress.
```

Comment:
what does it mean "PR is in progress"? 
I think the distinction between WIP and Draft needs to be clearer. For example, WIP may indicate that the content is incomplete but what exists is correct, whereas Draft would indicate that the content is not organized, has not been verified, and may contain errors.

Reply:
WIP and Draft are synonyms in this context. Should I make this clear?

Reply:
@CsatariGergely Or just delete one of them including from the "Labels" list. Thanks

Reply:
Corrected in [c8c328e](https://github.com/cntt-n/CNTT/pull/2805/commits/c8c328ea479ffbf74fb060e8e8e7ad8630a0f4d2)